src/Diagrams/Core/Names.hs

```diff
@@ -1,11 +1,12 @@
+{-# LANGUAGE CPP                        #-}
```

Comment:
I'm pretty sure I added CPP to selectively turn off OverlappingInstances, and then found out that the module didn't need the latter.